### ISSUE TITLE
Differences between protein fitness models can be used to design variants of altered specificity

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#561]
**Authors:** Samuel P. Berry; Rachelle Gaudet; Debora S. Marks

**Venue:** bioRxiv
**Date:** 2026-06-14

**Relevance:** 0.940426

## Identifiers

- DOI: [10.64898/2026.06.10.731299](https://doi.org/10.64898/2026.06.10.731299)
- Source: biorxiv `10.64898/2026.06.10.731299`
- Paper: https://www.biorxiv.org/content/10.64898/2026.06.10.731299v1

## Abstract

The vastness of sequence space makes it challenging for directed evolution to efficiently traverse the fitness landscape. In recent years, unsupervised probabilistic models trained on natural sequences have shown promise for predicting the functional effects of mutations and designing new proteins, leading many to suggest that these models may be useful for guiding directed evolution campaigns toward functional regions of sequence space. However, many directed evolution campaigns are interested in evolving new substrate or ligand specificity, and the behavior of unsupervised sequence models on predicting and designing activity against altered substrates or ligands has not been tested. We have built a curated database of multiplexed functional assay results profiling substrate or ligand specificity and systematically how assessed various popular unsupervised protein fitness models score and design variants that alter selectivity in these datasets. We find that sequence models that learn from the surrounding sequence context, especially protein language models, systematically bias against variants that alter specificity. This bias leads them to design altered-specificity variants at similar or lower rates than random chance. However, we propose a simple strategy to exploit this bias by taking weighted differences of model scores to enrich libraries for altered-specificity variants. These findings and our database should help guide how biologists best use protein fitness models and provide a framework to help machine learning researchers develop a new generation of machine learning models that can better design novelty.

## BibTeX

The collision-checked citation key is `berry2026`; the entry below is ready to copy.

```bibtex
@misc{berry2026,
  title = {Differences between protein fitness models can be used to design variants of altered specificity},
  author = {Samuel P. Berry and Rachelle Gaudet and Debora S. Marks},
  year = {2026},
  abstract = {The vastness of sequence space makes it challenging for directed evolution to efficiently traverse the fitness landscape. In recent years, unsupervised probabilistic models trained on natural sequences have shown promise for predicting the functional effects of mutations and designing new proteins, leading many to suggest that these models may be useful for guiding directed evolution campaigns toward functional regions of sequence space. However, many directed evolution campaigns are interested in evolving new substrate or ligand specificity, and the behavior of unsupervised sequence models on predicting and designing activity against altered substrates or ligands has not been tested. We have built a curated database of multiplexed functional assay results profiling substrate or ligand specificity and systematically how assessed various popular unsupervised protein fitness models score and design variants that alter selectivity in these datasets. We find that sequence models that learn from the surrounding sequence context, especially protein language models, systematically bias against variants that alter specificity. This bias leads them to design altered-specificity variants at similar or lower rates than random chance. However, we propose a simple strategy to exploit this bias by taking weighted differences of model scores to enrich libraries for altered-specificity variants. These findings and our database should help guide how biologists best use protein fitness models and provide a framework to help machine learning researchers develop a new generation of machine learning models that can better design novelty.},
  doi = {10.64898/2026.06.10.731299},
  url = {https://doi.org/10.64898/2026.06.10.731299},
  note = {Version 1},
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **Context-aware protein fitness models systematically score specificity-altering variants below variants that preserve native specificity** [@berry2026]. Figures 2A–C and 6, Tables 1–2, and the Results and Discussion compare 14 zero-shot models across eight heterogeneous assays: PLMs and alignment-covariation models have the lowest altered-specificity indices, the context-free PSSM is approximately neutral, and ESM-IF1 shows relatively little bias. Evidence is moderate because the pattern spans model classes and proteins, but the study is a non-peer-reviewed preprint, specificity classes differ by assay, and ProteinMPNN was excluded after weak native/inactive separation. This sharpens the garden’s claims that PLMs struggle with functional novelty and that sequence data do not uniquely identify fitness; the model penalty need not indicate destabilization. A hierarchical analysis separating assay-labeling variance from model-class variance would test whether the class-level effect survives the heterogeneous specificity definitions. Related: [[Language models cannot extrapolate to functional novelty or ultra-high-fitness variants]]; [[Unbalanced composition of sequence data prevents protein fitness from being identifiable from sequence data alone]]; [[Mutations that give rise to new functions are not more destabilizing than mutations in general]]. Placement: new atomic note linked from [[Fitness prediction]].

- **Performance on native-function mutation benchmarks does not predict the ability to rank specificity-altering variants** [@berry2026]. Figure 3A–H, the “Context-aware fitness models bias against altered-specificity variants” Results section, and “Recommendations for protein engineers” show heterogeneous and sometimes inverse relationships between native-specificity/inactive ROC AUC and altered-specificity index. Evidence is moderate across eight datasets, but inactive labels may include activity on untested substrates and the two metrics depend on dataset-specific class definitions. This supplies a specificity-focused limitation for [[ProteinGym]], agrees with the absence of a universally best zero-shot fitness predictor, and parallels failures of common model scores to predict enzyme activity. A calibrated three-class evaluation could reveal when conventional benchmarks reward models for treating altered-but-functional variants as errors, rather than summarizing that behavior with separate AUC and ASI axes. Related: [[ProteinGym]]; [[No one-size-fits-all best approach to zero-shot or few-shot protein fitness prediction]]; [[Most ML quality metrics cannot effectively predict enzyme activity after controlling for similarity to native]]. Placement: add to [[Language models cannot extrapolate to functional novelty or ultra-high-fitness variants]].

- **Protein language models enrich specificity-altering variants no better than—and often worse than—random sampling in retrospective simulated design** [@berry2026]. Figure 4A–E and the “Protein language models mostly design single variants with native specificity” Results section show that several PLMs and covariation models return no altered-specificity variants among their highest-ranked 5% for some datasets; the PSSM is neutral on average and inverse-folding models are slightly better but highly variable and difficult to distinguish from random. Evidence is strong within the assembled benchmark but limited to retrospective ranking of already measured variants rather than prospective design, with heterogeneous assay scales and mostly single substitutions. This directly supports the existing functional-novelty limitation; sequence divergence in PLM designs does not imply altered specificity. Distance-matched random baselines would separate a specificity-specific failure from a generic preference for near-native sequences. Related: [[Language models cannot extrapolate to functional novelty or ultra-high-fitness variants]]; [[PLMs generate more divergent functional proteins than Potts models]]; [[Protein language models are better zero-shot predictors for ranking closely related sequences than distantly related sequences]]. Placement: new atomic note linked from [[Fitness prediction]].

- **Disagreement between context-free and context-aware fitness scores can enrich retrospective libraries for specificity-altering variants** [@berry2026]. Figure 5A–F and Methods subsection “Fitting weighted difference models” show that every PSSM pairing receives a zero or negative contribution from the context-aware model; leave-one-dataset-out evaluation yields approximately 2.5-fold mean enrichment across all datasets and approximately fourfold enrichment for the five datasets representing acquisition of non-native activity. Evidence is moderate because held-out evaluation reduces direct leakage, but only eight heterogeneous datasets were available, the result is entirely retrospective, and no prospective experimental library was tested. Unlike additive logit averaging for conventional fitness prediction, altered specificity benefits from model disagreement; this reinforces task dependence among zero-shot predictors. Disagreement may detect out-of-distribution or family-mode mismatch rather than specificity itself, requiring controls against destabilization and other non-specific functional changes. Related: [[Averaging logits from multiple sources can improve fitness prediction]]; [[No one-size-fits-all best approach to zero-shot or few-shot protein fitness prediction]]; [[Unbalanced composition of sequence data prevents protein fitness from being identifiable from sequence data alone]]. Placement: new atomic note linked from [[Fitness prediction]].

#### Metadata check

The issue BibTeX and DOI registry agree on title, authors, year, and DOI `10.64898/2026.06.10.731299`. The issue uses `@misc{berry2026}` with `note = {Version 1}`, whereas DOI content negotiation returns an `@article` record with publisher `openRxiv` and month `June`; retaining the issue citekey is appropriate. Neither `berry2026` nor the DOI currently occurs in `bibliography.bib`. The official 37-page version 1 PDF was inspected in full, including all main figures and tables. bioRxiv separately links “Supplemental information (Supplemental Notes 1–2 and supplemental figures),” but its attachment server returned rate-limit error 1015; the supplement could not be independently inspected and is not relied upon above.
